### DISCUSSION TITLE
Can finch be used for CDK lambda bundling?

### DISCUSSION THREAD

**Discussion by skurihara-cw** [runfinch/finch#336] [Q&A] [answered]
If possible, please give me a sample code.

I would like to know if it works with code like this link.
https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda-readme.html#bundling-asset-code
My understanding is that docker is running internally.

Example with Python:
```
new lambda.Function(this, 'Function', {
  code: lambda.Code.fromAsset(path.join(__dirname, 'my-python-handler'), {
    bundling: {
      image: lambda.Runtime.PYTHON_3_9.bundlingImage,
      command: [
        'bash', '-c',
        'pip install -r requirements.txt -t /asset-output && cp -au . /asset-output'
      ],
    },
  }),
  runtime: lambda.Runtime.PYTHON_3_9,
  handler: 'index.handler',
});
```

**Comment by ningziwen** [comment, marked as answer]
It is WIP. There is an issue tracking the work of Finch CDK integration. https://github.com/aws/aws-cdk/issues/23680
